app/client/src/routes/home.tsx

```diff
@@ -266,6 +295,8 @@ export function QueryBuilder() {
           </AccordionItem>
 
           <AccordionItem heading="Advanced API Queries">
+            Visit our <Link to="/api-documentation">API Documentation</Link>{' '}
```

Comment:
You will lose your filters if you click on this link. Can you make this open in a new tab and add a link back to attains query page on the api documentation page?